### DISCUSSION TITLE
Welcome to b2evolution Discussions!

### DISCUSSION THREAD

**Discussion by fplanque** [b2evolution/b2evolution#124] [Announcements]
As the b2evolution forums run in limited mode, I am opening this space to allow further discussion, if needed.

**Comment by soundos28** [comment]
your site is very useful
<a href="[https://www.univ-msila.dz/site/](https://www.univ-msilla.dz/site/)segc /">SOUNDOS-SEGC</a>.

**Comment by Skipperdan76** [comment]
Hello, I am not sure I am in the correct spot. If not maybe you can direct me to a place that can help.
I just took over a site on GoDaddy with another partner. I want to add a blog to the site, or multiple blogs. I think b2evolution would work. I am not sure if I can implement it or my partner would have to do it, as he will be the owner of the GoDaddy account. I am just an administrator of our site. 
Any help you could give me would be appreciated, 
Daniel, skipperdan76@gmail.com
